DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -288,6 +289,24 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
             .store(in: &cancellables)
     }
 
+    private func setUpErrorPublishers() {
+        guard AppDependencyProvider.shared.internalUserDecider.isInternalUser else {
+            return
+        }
+
+        errorObserver.publisher
+            .map { errorMessage in
+                guard let errorMessage else {
+                    return nil
+                }
+
+                return ErrorItem(title: "Failed to Connect", message: errorMessage)
```

Comment:
Hardcoded the copy here for now so that it doesn't get accidentally translated, we'll come back to this once the error UI is complete and has final copy - for now I just want to get the UI itself in place so that we can use it.